shopfloor_mobile_base/static/wms/src/services/config_registry.js

```diff
@@ -20,10 +20,23 @@ class StoredConfig {
         this.key = key;
         this.default = meta.default;
         this.reset_on_clear = meta.reset_on_clear;
+        if (!_.isEmpty(meta.storage)) {
```

Comment:
Do not change the original value. Just raise an error if you try to register a wrong type.
This should not pass silently

Reply:
Added.